spec/02-integration/09-hybrid_mode/09-config-compat_spec.lua

```diff
@@ -635,6 +635,28 @@ describe("CP/DP config compat transformations #" .. strategy, function()
       end)
     end)
 
+    describe("prometheus plugins", function()
+      it("[prometheus] remove ai_metrics property for versions below 3.8", function()
+        -- [[ 3.7.x ]] --
```

Comment:
```suggestion
        -- [[ 3.8.x ]] --
```
as the following is config for 3.8